components/ipfs/ipfs_service.cc

```diff
@@ -320,23 +325,18 @@ bool IpfsService::IsDaemonLaunched() const {
 }
 
 void IpfsService::LaunchDaemon(LaunchDaemonCallback callback) {
-  // Reject if previous launch request in progress.
-  if (launch_daemon_callback_) {
-    std::move(callback).Run(false);
-    return;
-  }
-
-  if (allow_ipfs_launch_for_test_) {
+  if (IsDaemonLaunched()) {
```

Comment:
I think this is including a commit from the other PR.

Reply:
yes, branched it from previous https://github.com/brave/brave-core/pull/7778/files, diff will disappear after merge 7778, but I can create a new branch from master